tests/misc_env/push_cosbench_workload.py

```diff
@@ -98,27 +127,27 @@ def prepare_fill_workload(ceph_cluster, client, rgw, controller, config):
         port = rgw_service[0]["status"]["ports"][0]
         workload_endpoint = f"http://{ip}:{port}"
     LOG.info(f"workload endpoint: {workload_endpoint}")
-    fill_workload = fill_workload.replace("workload_endpoint", workload_endpoint)
+    workload_conf = workload_conf.replace("workload_endpoint", workload_endpoint)
 
     out, err = controller.exec_command(
         cmd="sh /opt/cosbench/cli.sh info | grep drivers | awk '{print $2}'"
     )
     LOG.info(out)
     drivers_count = int(out.strip())
     workers = drivers_count * 100
-    fill_workload = fill_workload.replace(
+    workload_conf = workload_conf.replace(
         'work type="prepare" workers="1"', f'work type="prepare" workers="{workers}"'
     )
-
-    workload_file_id = utils.generate_unique_id(length=4)
-    workload_file_name = f"fill-workload-{workload_file_id}.xml"
-    fill_workload = fill_workload.replace(
-        "pri-bkt", f"pri-bkt-{workload_file_id.lower()}-"
+    workload_conf = workload_conf.replace(
+        'work type="hybrid" workers="1"', f'work type="hybrid" workers="{workers}"'
```

Comment:
@anrao, in the hybrid workload logs workers is still showing 1
http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/cephci-run-TPBLK8/push_cosbench_hybrid_workload_0.log
but for fill workload test, workers are 200
could you please check?

Reply:
Updated the pr @hmaheswa